packages/mux-player/src/index.ts

```diff
@@ -593,6 +604,78 @@ class MuxPlayerElement extends VideoApiElement {
     return this.media?.mux;
   }
 
+  /**
+   * Get Mux asset playback id.
+   */
+  get playbackId() {
+    // Don't get the mux-video attribute here because it could have the
+    // playback token appended to it.
+    return this.getAttribute(MuxVideoAttributes.PLAYBACK_ID) ?? undefined;
+  }
+
+  /**
+   * Set Mux asset playback id.
+   */
+  set playbackId(val) {
+    if (val) {
+      this.setAttribute(MuxVideoAttributes.PLAYBACK_ID, val);
+    } else {
+      this.removeAttribute(MuxVideoAttributes.PLAYBACK_ID);
+    }
+  }
+
+  /**
+   * Get the string that reflects the src HTML attribute, which contains the URL of a media resource to use.
+   */
+  get src() {
+    // Only get the internal video.src if a playbackId is present.
+    if (this.playbackId) {
+      return getVideoAttribute(this, VideoAttributes.SRC) ?? undefined;
+    }
+    return this.getAttribute(VideoAttributes.SRC) ?? undefined;
+  }
+
+  /**
+   * Set the string that reflects the src HTML attribute, which contains the URL of a media resource to use.
+   */
+  set src(val) {
+    if (val) {
+      this.setAttribute(VideoAttributes.SRC, val);
+    } else {
+      this.removeAttribute(VideoAttributes.SRC);
+    }
+  }
```

Comment:
sorry I moved these blocks, they are moved more to the top because of importance. didn't change logic